pkg/installmanager/installmanager.go

```diff
@@ -959,6 +959,7 @@ func uploadAdminKubeconfig(provision *hivev1.ClusterProvision, m *InstallManager
 		},
 	}
 
+	controllerutils.SetOwnerLabel(provision, kubeconfigSecret)
```

Comment:
Not a concern for this PR, but this does bring to light a bit of an issue. I think that we want the ownership of the kubeconfig and admin password secrets to be passed from the ClusterProvision to the ClusterDeployment once the provision has completed.